x-pack/plugin/esql/qa/server/single-node/src/yamlRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/AbstractEsqlClientYamlIT.java

```diff
@@ -44,4 +50,40 @@ private void assertRequestBreakerEmpty() throws Exception {
          */
         EsqlSpecTestCase.assertRequestBreakerEmpty();
     }
+
+    public static Iterable<Object[]> updateEsqlQueryDoSections(Iterable<Object[]> parameters, Function<DoSection, ExecutableSection> modify)
```

Comment:
Refactored `x-pack/plugin/esql/qa/server/single-node/src/yamlRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/EsqlClientYamlAsyncIT.java` a little bit, but failed moving this into a place to be used by multiple yaml tests from :/